packages/kms-local/src/key-management-system.ts

```diff
@@ -130,6 +130,8 @@ export class KeyManagementSystem extends AbstractKeyManagementSystem {
         return await this.eth_signMessage(managedKey.privateKeyHex, data)
       } else if (['eth_signTypedData', 'EthereumEip712Signature2021'].includes(algorithm)) {
         return await this.eth_signTypedData(managedKey.privateKeyHex, data)
+      } else if (['eth_rawSign'].includes(algorithm)) {
```

Comment:
This KMS should also advertise this algorithm on the `meta.algorithms` property of the keys it creates.
Please see the `asManagedKeyInfo` method below.

At some point we will want to have finer control over the KMS chosen for particular operations based on the key capabilities (`meta.algorithms`), so I think it's worth keeping these 2 in sync.